src/liquid/components/ld-select/test/ld-select.spec.ts

```diff
@@ -2008,4 +2008,26 @@ describe('ld-select', () => {
     await page.root.focusInner()
     expect((btnTrigger as HTMLElement).focus).toHaveBeenCalledTimes(1)
   })
+
+  it('auto-focuses', async () => {
+    const page = await newSpecPage({
+      components,
+      html: `
+        <ld-select autofocus placeholder="Pick a fruit" name="fruit" popper-class="ld-select__popper--fruits">
+          <ld-option value="apple">Apple</ld-option>
+          <ld-option value="banana">Banana</ld-option>
+        </ld-select>
+      `,
+    })
+
+    const ldSelect = page.root
+    const btnTrigger = ldSelect.shadowRoot.querySelector(
+      '.ld-select__btn-trigger'
+    )
+    ;(btnTrigger as HTMLElement).focus = jest.fn(focusManager.focus)
+
+    await page.waitForChanges()
+    jest.advanceTimersByTime(0)
+    expect((btnTrigger as HTMLElement).focus).toHaveBeenCalledTimes(1)
```

Comment:
The focusManager does not really do anything here to support the test case. Wouldn't use it then.

```suggestion
    const btnTrigger: HTMLElement = ldSelect.shadowRoot.querySelector(
      '.ld-select__btn-trigger'
    )
    btnTrigger.focus = jest.fn()

    await page.waitForChanges()
    jest.advanceTimersByTime(0)
    expect(btnTrigger.focus).toHaveBeenCalledTimes(1)
```

Reply:
removed